### ISSUE TITLE
Current (active) slide should have an indicator class applied

### ISSUE THREAD

**Issue by zdolin** [FormidableLabs/nuka-carousel#914]
# Bugs and Questions

### Describe Your Environment

- What version of nuka-carousel are you using? v5.1.2
- What version of React are you using? React 17.0.2 (also 18.0)
- What browser are you using? Chrome

### Describe the Problem

Codesandbox: 
https://codesandbox.io/s/cranky-sea-j5uszw

**Expected behavior:** Current (active) slide should have an indicator class applied (".slide-current" or similar).
**Actual behavior:** No special classes are applied to the current slide besides ".slide .slide-visible", which doesn't distinguish it from any of the others.  Therefore, there's no way to tell programmatically (from what I can tell) what the current slide is, or apply special styling for the active/current slide.

Also note that wrapAround behavior is buggy when scrolling left (observable when clicking Prev a few times).

Thanks!

**Comment by zdolin** [comment]
Note this occurs when using the following combination of props (see codesandbox above):
```
{
  wrapAround={true}
  animation="zoom"
  slidesToShow={6}
  cellAlign="center"
}
```
I think the key issue is `slidesToShow` being greater than 1, which means multiple slides are visible at once and therefore it cannot be distinguished which is the active slide.

**Comment by ValGeorgiev** [comment]
Hey @zdolin thanks for raising this with us. It sounds easy to fix, I will prepare something today 🤞

**Comment by zdolin** [comment]
@ValGeorgiev great, thank you so much! Did you see my note about the wrapAround behavior as well? I can log a separate ticket for that, if you'd prefer.

**Comment by ValGeorgiev** [comment]
@zdolin Yes, another issue with sandbox will do the job. Thanks

**Comment by ValGeorgiev** [comment]
This should be fixed in v5.1.3
